libvcx/src/init.rs

```diff
@@ -30,11 +30,15 @@ struct IssuerConfig {
     institution_verkey: Option<String>,
 }
 
-pub fn init_core(config: &str) -> VcxResult<()> {
-    info!("init_core >>> config = {}", config);
-    settings::process_config_string(&config, true)?;
-    settings::log_settings();
-    init_threadpool(config)?;
+pub fn enable_vcx_mocks() -> VcxResult<()> {
+    info!("enable_vcx_mocks >>>");
+    settings::set_config_value(settings::CONFIG_ENABLE_TEST_MODE, "true");
+    Ok(())
+}
+
+pub fn enable_agency_mocks() -> VcxResult<()> {
+    info!("enable_agency_mocks >>>");
+    settings::get_agency_client_mut()?.enable_test_mode();
     Ok(())
```

Comment:
Where are these used?

Reply:
NodeJS tests, previously NodeJS unit tests were using `vcx_init_core` where was passed configuration field to enable mocks. As `vcx_init_core` was removed, I've added these functions as a way for NodeJS to tell aries-vcx that mocks should be enabled.